lib/dep-types.ts

```diff
@@ -22,16 +25,19 @@ var depTypes = module.exports = function (depName, pkg) {
   }
 
   var bundled = !!(pkg.bundleDependencies &&
-    pkg.bundleDependencies.indexOf(depName) !== -1);
+    pkg.bundleDependencies[depName]);
 
   return {
-    type: type,
+    type: type as string,
     from: from,
     bundled: bundled,
```

Comment:
just `bundled,` can use shorthand

Reply:
Intentionally omitted to keep the diff short.